sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -75,7 +72,6 @@ private[hive] object IsolatedClientLoader extends Logging {
               "again. Hadoop classes will not be shared between Spark and Hive metastore client. " +
```

Comment:
Shall we update this message since we now always share? Actually we should think about throwing an exception that says all related metastore jars have to be specified instead.

Reply:
Forgot about this place - will fix. Yes agree that an exception might be more appropriate. It seems a bit arbitrary to fall back to 2.7.4 here since the original Hadoop version could vary based on the Hadoop profile (either 2.7.4 or 3.2.0 at the moment).